.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @confluentinc/clients
+*	@confluentinc/clients @confluentinc/data-governance
```

Comment:
Is the spacing in front intentional?

Reply:
Taken from a different file, it's a tab instead of a space